### DISCUSSION TITLE
Payload CMS with NGINX in Production with HTTPS

### DISCUSSION THREAD

**Discussion by xr71** [payloadcms/payload#828] [Q&A] [answered]
I have followed all of the steps in the Digital Ocean guide and have updated my `serverURL` but I am still getting a 404 when I attempt to route `/admin`

I have set up nginx and certbot with letsencrypt, but I cannot access the frontend admin pannel.  Thanks for all your help!

**Comment by MinSomai** [comment]
can you post your nginx conf. if possible.

**Comment by xr71** [comment, marked as answer]
Sorry, dumb mistake, just fixed it; I had to add the proxy block in nginx.conf for /admin to pass to localhost:3000/admin.  Thanks!
